src/Form/Themes.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Handle Form Themes
+ *
+ * @package Give
+ * @since 2.7.0
+ */
+
+namespace Give\Form;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Themes class
+ *
+ * @since 2.7.0
+ */
+class Themes {
+	/**
+	 * Themes
+	 *
+	 * @var array
+	 */
+	private $themes = array();
+
+
+	/**
+	 * Get Registered themes
+	 *
+	 * @since 2.7.0
+	 *
+	 * @return array
+	 */
+	public function get() {
+		return $this->themes;
+	}
+
+	/**
+	 * Get Registered theme
+	 *
+	 * @since 2.7.0
+	 *
+	 * @param string $themeID
+	 *
+	 * @return Theme
+	 */
+	public function getTheme( $themeID ) {
```

Comment:
This is really the point at which you could load the theme. Just have the `$themeID` be the same (or something you can derive to) the file name, then load the theme _as needed_.

Reply:
this will be resolve in https://github.com/impress-org/givewp/pull/4518